Xamarin.PropertyEditing/Xamarin.PropertyEditing.csproj

```diff
@@ -15,4 +15,9 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Do we need this, given the above? cc @alanmcgovern

Reply:
@ermau That was auto added by the IDE.

Reply:
This should be fine. It's just ensuring the autogenerated file shows up as a child of the 'resx' file in the solution pane. If we don't add it the odds are the IDE will continually try to re-add it.